test/integration/targets/lambda_info/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/aws
+shippable/aws/group2
+lambda
```

Comment:
There's already an alias for the lambda tests in the `aws_lambda` test suit, it's easy to miss since we don't have an aws_lambda module.  We should probably combine these tests with that existing set of tests, that will save us having to have multiple copies of mini-lambda.py.

Reply:
So should I remove this line, or change it to `aws_lambda`?

Reply:
Instead of making a new lambda_info test suite, merge the tests for the `lambda_info` module into the `targets/aws_lambda` test suite and add `lambda_info` to `aws_lambda/aliases`.  The `ec2_vpc_net` tests are a good example of a module and it's corresponding *_info module being in the same test suite.

Reply:
Ah, got it now, was just confused earlier by your inline comment. Will move tests to existing lambda testsuite now.